packages/nextjs/src/api/index.ts

```diff
@@ -1,4 +1,17 @@
-export { default as Clerk } from '@clerk/clerk-sdk-node';
+import { Clerk } from '@clerk/backend';
+
+import { clerkClient } from '../server';
+
+const createClerkClient = Clerk;
+
+export { createClerkClient };
+
+export * from '@clerk/backend';
+
 export { requireAuth } from './requireAuth';
 export { withAuth } from './withAuth';
-export * from '@clerk/clerk-sdk-node';
+
+const { users, smsMessages, sessions, emailAddresses, phoneNumbers, emails, invitations, organizations, clients } =
```

Comment:
In V5 we need to revise this as I'd like ideally to have a common way of exporting this across our SDKS and since Remix doesn't allow us to do it, maybe, we should stick to the createClerkClient approach. Let's defer the conversation for the V5 release.